packages/editor/api-report.md

```diff
@@ -845,7 +867,7 @@ export class Editor extends EventEmitter<TLEventMap> {
     packShapes(ids: TLShapeId[], gap: number): this;
     get pages(): TLPage[];
     get pageStates(): TLInstancePageState[];
-    pageToScreen(x: number, y: number, z?: number, camera?: VecLike): {
+    pageToScreen(x: number, y: number, z?: number): {
```

Comment:
camera was unused as a prop

Reply:
These conversions are small enough that they could be done inline if you really needed to convert a page point for a different page's camera.